src/datastructures.apib

```diff
@@ -140,3 +140,121 @@
         + timeTracking.deleted
         + timeTracking.updated
         + user.deactivated
+
+## WorkOrderData (object)
++ customer (object, required)
+    + type (enum[string], required)
+        + Members
+            + contact
+            + company
+    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
++ datetime: `2017-05-26T10:01:49+00:00` (string, required)
++ description (string, optional)
++ department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional)
++ layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional)
++ milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
++ event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
++ intervention_address (object, optional)
+    + One Of
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + address (Address)
++ distance: `25.2` (number, optional)
++ users (array, required)
+    + (object)
+        + user_id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string, required)
+        + hourly_rate: `65` (number, optional)
+        + work_hours (array, optional)
+            + (object)
+                + One Of
```

Comment:
Q : do we need to provide both ways?

Reply:
@JorenSaey what do you prefer here? Do you use a reusable component for creating and editing time trackings that supports both formats, or does one format suffice?

Reply:
I prefer just the sarted_at and duration combo, that is what we use in our TimeTracking module. :)